js/index.js

```diff
@@ -36,7 +36,108 @@ const siteContent = {
     "copyright" : "Copyright Great Idea! 2018"
   },
 };
+//
+var log = document.querySelector('body');
 
+window.addEventListener('load', (event) => {
```

Comment:
I love the load animation.  It was weird though, I only got it to work sometimes.  I have no idea why.